docs/utilities/data_classes.md

```diff
@@ -1161,6 +1161,26 @@ You can register your Lambda functions as targets within an Amazon VPC Lattice s
     --8<-- "examples/event_sources/src/vpc_lattice_payload.json"
     ```
 
+### Secrets Manager
+
+=== "app.py"
+
+    ```python
+    from aws_lambda_powertools.utilities.data_classes import event_source, SecretManagerEvent
+
+    @event_source(data_class=SecretManagerEvent)
+    def lambda_handler(event: SecretManagerEvent, context):
+        # Multiple records can be delivered in a single event
+        service_client = boto3.client('secretsmanager', endpoint_url=os.environ['SECRETS_MANAGER_ENDPOINT'])
+        create_secret(service_client, event.secret_id, event.client_request_token)
+        ...
```

Comment:
What do you think about making this example more realistic? It's missing the boto3 import, the `create_secret` function is missing, and the comment is also wrong. Can you improve a little?

Reply:
this is a example from https://github.com/aws-samples/aws-secrets-manager-rotation-lambdas, but they seems too complicated for a simple example. I extracted `service_client.get_secret_value` to demo our case here. Please help to check if this looks good to you. Thanks!

Reply:
Hey @roger-zhangg! I changed some lines in the example. Please check it and see if you agree.

Reply:
Thanks for the fix, I'm not familiar to parameter so thanks for pointing it out!